dali/operators/reader/loader/coco_loader.h

```diff
@@ -59,6 +64,35 @@ inline bool HasSavePreprocessedAnnotationsDir(const OpSpec &spec) {
     (spec.HasArgument("dump_meta_files_path") && spec.GetArgument<bool>("dump_meta_files_path"));
 }
 
+struct RLEMask : public UniqueHandle<RLE, RLEMask> {
+  DALI_INHERIT_UNIQUE_HANDLE(RLE, RLEMask)
+
+  RLEMask(siz h, siz w, siz m) {
+    rleInit(&handle_, h, w, m, nullptr);
+  }
+
+  RLEMask(siz h, siz w, span<const uint> counts) {
+    rleInit(&handle_, h, w, counts.size(), const_cast<uint*>(counts.data()));
+  }
+
+  RLEMask(siz h, siz w, const char* str) {
+    rleFrString(&handle_, const_cast<char*>(str), h, w);
+  }
+
+  static constexpr RLE null_handle() { return {0, 0, 0, nullptr}; }
```

Comment:
Do you need this? Won't the default just work?

Reply:
I remember it didn't work (ptr was not nullptr, if I remember correctly). I could double-check

Reply:
It seems to work, so I am removing it